modules/light-clients/08-wasm-light-clients/Cargo.toml

```
@@ -0,0 +1,39 @@
+[workspace]
+resolver = "2"
+members = [
+    "contracts/*",
+    "packages/*",
```

Instead of using `*`, you probably want to point to the exact directory instead since this is a multi language repo.

```suggestion
    "contracts/ethereum-light-client",
    "packages/ibc-go-proto",
```

---

It's a multi language repo, but the folder we're in is for the rust code. If we suddenly add anything there, builds will fail and then we can adjust, no?